bedrock/mozorg/templates/mozorg/home/includes/m24/ai-gallery.html

```diff
@@ -224,10 +224,10 @@ <h2 class="m24-c-intro-title">{{ ftl('m24-home-join-the-movement') }}</h2>
             "alt": ftl('m24-home-alt-llamafile-logo'),
           }
         ),
-        tag=ftl('m24-home-tag-product'),
+        tag=ftl('m24-home-tag-project', fallback='m24-home-tag-project'),
```

Comment:
The fallback is the same ID, I think you meant it to fall back to `product`.
But maybe it doesn't even need a fallback. You could hide the whole tag if the string isn't available, but it could also default to English since it's just a one word tag?

Reply:
```suggestion
        tag=ftl('m24-home-tag-project', fallback='m24-home-tag-product'),
```